### PR TITLE
fix: rendering of tokens on mobile tokenselector

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Addresses https://github.com/cowprotocol/cowswap/issues/4927 
- Also hides the balance element if no account is connected/detected
- Center align small limit order banner title

Screenshot 2024-09-30 at 14 12 26
Screenshot 2024-09-30 at 14 12 04

Screenshot 2024-09-30 at 14 40 27

### CODE REVIEW

**Review by elena-zh — CHANGES_REQUESTED** [review]
Hey @fairlighteth , thank you for addressing this!
The issue with the warning's name is fixes. 
However, still there are some issues with token lists: 
1. Not connected wallet: 
image
2. Issue with a token address on a small screen width is still here: 
image

Could you please take a look at these cases?

**Review by elena-zh — APPROVED** [review]

**Comment by elena-zh** [comment]
Fixed now, thank you!

File: apps/cowswap-frontend/src/modules/tokensList/pure/TokensVirtualList/index.tsx [hunk 1]
```diff
@@ -31,6 +32,9 @@ export function TokensVirtualList(props: TokensVirtualListProps) {
   const { allTokens, selectedToken, balancesState, onSelectToken, unsupportedTokens, permitCompatibleTokens } = props
   const { values: balances, isLoading: balancesLoading } = balancesState
 
+  const { account } = useWalletInfo()
```
Comment (shoom3301) [inline apps/cowswap-frontend/src/modules/tokensList/pure/TokensVirtualList/index.tsx:35]: This is a pure component, it's not supposed to contain external dependencies.
The account should be passed through props

Comment (fairlighteth) [reply apps/cowswap-frontend/src/modules/tokensList/pure/TokensVirtualList/index.tsx]: @shoom3301 I have addressed your comment now

**Review by shoom3301 — APPROVED** [review]

**Pull request merged**